lib/hls/hls_parser.js

```diff
@@ -888,6 +888,27 @@ shaka.hls.HlsParser = class {
     return count;
   }
 
+  /**
+   * Get the spatial audio information for an HLS audio track.
+   * In HLS the channels field indicates the number of audio channels that the
+   * stream has (eg: 2). In the case of Dolby Atmos, the complexity is
+   * expressed with the number of channels followed by the word JOC
+   * (eg: 16/JOC), so 16 would be the number of channels (eg: 7.3.6 layout),
+   * and JOC indicates that the stream has spatial audio.
+   * @see https://developer.apple.com/documentation/http_live_streaming/hls_authoring_specification_for_apple_devices/hls_authoring_specification_for_apple_devices_appendixes
+   *
+   * @param {!shaka.hls.Tag} tag
+   * @return {boolean}
+   * @private
+   */
+  getSpatialAudio_(tag) {
+    const channels = tag.getAttributeValue('CHANNELS');
+    if (!channels) {
```

Comment:
nit: return channels && channels.includes('/JOC');

Reply:
This produces an error:

```
[WARNING] No changes detected, skipping. Use --force to override.
/Users/alvaro.velad/workspace/pullrequest-shaka-player/lib/hls/hls_parser.js:906:11: ERROR - [JSC_TYPE_MISMATCH] inconsistent return type
found   : (boolean|null|string)
required: boolean
  906|     return channels && channels.includes('/JOC');
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1 error(s), 0 warning(s), 94.5% typed
[ERROR] Build failed

                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```